src/modules/acceptance-tests/controller.js

```diff
@@ -2,7 +2,32 @@ const db = require('../../lib/connectors/db');
 const ACCEPTANCE_TEST_SOURCE = 'acceptance-test-setup';
 const config = require('../../../config');
 
+const deleteUserRoles = () => {
```

Comment:
I'm not sure if this is necessary as both user_groups and user_roles have on delete cascade set up.
I tried deleting a user locally and its group and role were cleaned up automatically.